docs/rules/group-exports.md

```diff
@@ -0,0 +1,62 @@
+# group-exports
+
+Reports when multiple named exports or CommonJS assignments are present in a single file and when the default export is not adjacent to the named export.
+
+**Rationale:** An `export` declaration or `module.exports` assignment can appear anywhere in the code. By requiring a single export declaration along with the default export being placed next to the named export, your exports will remain at one place, making it easier to see what exports a module provides.
+
+## Rule Details
+
+This rule warns whenever a single file contains multiple named exports or assignments to `module.exports` (or `exports`) and when the default export is not adjacent to the named export.
+
+### Valid
+
+```js
+// Default export is adjacent to named export -> ok
+export default function test() {}
+// A single named export -> ok
+export const valid = true
+```
+
+```js
+const first = true
+const second = true
+
+// A single named export -> ok
```

Comment:
technically this is a single named export *statement* but it's two named exports - can we add "statement"?